apps/agent/app/src/components/Transactions.js

```diff
@@ -57,8 +56,13 @@ const Transactions = React.memo(function Transactions({
     selectedToken,
     selectedTransactionType,
     symbols,
+    transactionTypes,
   } = useFilteredTransactions({ transactions, tokens })
 
+  const transactionLabels = useMemo(() => Object.values(transactionTypes), [
+    transactionTypes,
+  ])
```

Comment:
Related to previous – If we decide to import `TRANSACTION_TYPES_LABELS` in `<TransactionFilters/>` we could move this there as well.

Reply:
Yup; definitely not the prettiest thing there, but the reasoning is stated above. :)